trunk/SUAVE/Plots/Airfoil_Plots.py

```diff
@@ -0,0 +1,151 @@
+## @ingroup Plots
+# Airfoil_Plots.py
+#
+# Created: Mar 2021, M. Clarke
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+import SUAVE
+from SUAVE.Core import Units
+import numpy as np 
+import matplotlib.pyplot as plt  
+import matplotlib.cm as cm
+
+## @ingroup Plots
+def plot_airfoil_batch_properties(ap,arrow_color = 'r',plot_pressure_vectors = False ):  
```

Comment:
docstring

Reply:
I will actually separate this into three functions as well as add docstrings